modules/components/tag-input.ts

```diff
@@ -701,6 +707,74 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
         }
     }
 
+
+    /**
+     * @name DRAG_N_DROP_KEY
+     * @type {string}
+     */
+    private static DRAG_N_DROP_KEY: string = "dragged tag";
```

Comment:
I understand.. So I can create a separated class like "TagInputConstants" (maybe not class), to remove it from the component. And basically we can refactor something, if you sure that this component is enormous.